client/package.json

```diff
@@ -25,15 +25,12 @@
   "engines": {
     "vscode": "^1.75.0"
   },
-  "extensionDependencies": [
-    "mads-hartmann.bash-ide-vscode",
-    "ms-python.python"
-  ],
   "categories": [
     "Programming Languages"
   ],
   "activationEvents": [],
   "main": "./out/extension",
+  "browser": "./out/extension",
```

Comment:
It would be nice to not get an error on startup. I get this:
![image](https://github.com/yoctoproject/vscode-bitbake/assets/19970889/bc545838-fb8b-4c4f-a7a6-6fd527705739)
We have to define another browser, or adapt the extension code to not run imports for web.

Reply:
Sounds good :+1:

Reply:
Take a look at the new commit about the new entry point for browser